test/mastodon-tl-tests.el

```diff
@@ -1,4 +1,5 @@
 (require 'el-mock)
+(require 'cl-macs)
```

Comment:
I asked in the matrix channel... let's see what he thinks.

I personally don't think it's an issue.
As I understand it, use of the "undesirable" files is even considered palatable as long as they are only required for bytecompiling (i.e. macros are ok if they don't expand into something undesirable to be used at runtime).  Surely the same reasoning would apply for tests: we don't want to burden users to require these files for normal operation but for tests (or compiling) it's ok.